packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -6,7 +6,7 @@
       hidden: hidden }"
   >
     <router-link
-      v-if="entityType === types.link"
+      v-if="entityType === types.link && link.path && !link.path.toLowerCase().startsWith('http') && !link.path.toLowerCase().startsWith('www.')"
```

Comment:
entitiyType is dynamically calculated, let's leverage that functionality.
We can add smartLink type here https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/components/SidebarItem.vue#L121
And we can add checks for smartLink here https://github.com/okta/okta-developer-docs/blob/master/packages/%40okta/vuepress-theme-prose/components/SidebarItem.vue#L128
All entity type calculations will be in the same place which easier to maintain.

On the component level, we will have simple checks like `v-if="entityType === types.link"` and `v-if="entityType === types.smartLink"`

Reply:
Excellent point. 

I have updated the PR with the suggested changes.